GeneralsMD/Code/GameEngine/Source/GameNetwork/NetPacket.cpp

```diff
@@ -740,7 +803,9 @@ void NetPacket::FillBufferWithCommand(UnsignedByte *buffer, NetCommandRef *ref)
 			FillBufferWithGameCommand(buffer, ref);
 			break;
 		case NETCOMMANDTYPE_ACKSTAGE1:
+			FALLTHROUGH;
 		case NETCOMMANDTYPE_ACKSTAGE2:
+			FALLTHROUGH;
```

Comment:
Does not hurt, but is also not strictly required. By convention this fall through is implied. Static analyzers will not warn for these either.